tests/tests/control_p8_lndp

```diff
@@ -0,0 +1,135 @@
+###############################################################################
+#
+#  Global control test GFSv16 atmosphere only at C96L127, P8 configuration
+#
+###############################################################################
+
+export TEST_DESCR="Compare global control results with previous trunk version"
+
+export CNTL_DIR=control_p8_lndp
+
+export LIST_FILES="sfcf000.nc \
+                   sfcf021.nc \
+                   sfcf024.nc \
+                   atmf000.nc \
+                   atmf021.nc \
+                   atmf024.nc \
+                   GFSFLX.GrbF00 \
+                   GFSFLX.GrbF21 \
+                   GFSFLX.GrbF24 \
+                   GFSPRS.GrbF00 \
+                   GFSPRS.GrbF21 \
+                   GFSPRS.GrbF24"
+
+export_fv3
+export NPZ=127
+export NPZP=128
+export DT_ATMOS=720
+export SYEAR=2021
+export SMONTH=03
+export SDAY=22
+export SHOUR=06
+export RESTART_INTERVAL="12 -1"
+export OUTPUT_GRID='gaussian_grid'
+export WRITE_DOPOST=.true.
+export FHMAX=48
```

Comment:
Since only 0, 21 and 24h' history files are saved as baselines, there's no need to run 48h forecast.

Reply:
I'm going to change this to add testing of the 48 hr output, as I want the test to check that the global_cycle update at 24 hours (FHCYC) is handled properly.